packages/fast-form-generator-react/src/form/form.tsx

```diff
@@ -241,8 +241,8 @@ class Form extends React.Component<IFormProps & IManagedClasses<IFormClassNameCo
             }
         }
 
-        if (location.length >= 8 && location.slice(location.length - 8, location.length) === "children") {
-            const children: JSX.Element = get(obj, location);
+        if (isChildren) {
+            const children: any = get(obj, location);
```

Comment:
Is this actually "any"?

Reply:
see reply to rice, I'll update to be more precise